documentation/modules/auxiliary/admin/ldap/vmware_vcenter_vmdir_auth_bypass.md

```diff
@@ -0,0 +1,119 @@
+## Vulnerable Application
+
+### Description
+
+This module bypasses LDAP authentication in VMware vCenter Server's
+vmdir service to add an arbitrary administrator user. Version 6.7
+prior to the 6.7U3f update is vulnerable.
+
+### Setup
+
+Tested in the wild. No setup notes available at this time.
```

Comment:
Can we add a note to this to say setup is specific to target environment therefore we can't provide setup steps atm?